nucleus/cluster/admin/src/main/java/fish/payara/admin/cluster/AddInstanceToDeploymentGroupCommand.java

```diff
@@ -164,4 +172,25 @@ public void execute(AdminCommandContext context) {
         }
     }
 
-}
+    private String getVirtualServers(Server server) {
+        Config config = domain.getConfigs().getConfigByName(
+                server.getConfigRef());
+
+        StringJoiner virtualServers = new StringJoiner(",");
+        if (config != null) {
+            HttpService httpService = config.getHttpService();
+            if (httpService != null) {
+                List<VirtualServer> hosts = httpService.getVirtualServer();
+                if (hosts != null) {
+                    for (VirtualServer host : hosts) {
+                        if (("__asadmin").equals(host.getId())) {
+                            continue;
```

Comment:
It might be personal preference but I think use of `continue` should be avoided if possible. Here it is easy to avoid by negating the condition and moving the add into the block.